app/src/main/res/values-land/dimens.xml

```diff
@@ -384,10 +385,14 @@
   <dimen name="profile_rename_activity_profile_rename_input_margin_end">168dp</dimen>
   <dimen name="profile_rename_activity_profile_rename_save_button_margin_end">168dp</dimen>
 
+  <!-- Story Fragment- land dimens -->
+  <dimen name="story_fragment_activity_padding_bottom">60dp</dimen>
```

Comment:
The name cannot have `activity` as well as `fragment` please use only one based on which file is using this value.

Reply:
okk, I change it.